microsoft-edge/devtools-guide-chromium/webaudio/webaudio-tool.md

```diff
@@ -0,0 +1,26 @@
+---
+title: WebAudio tool
+description: About the WebAudio tool in Microsoft Edge DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/15/2022
+---
+# WebAudio tool
+
+Use the **WebAudio** tool to monitor **WebAudio** calls and traffic, for a webpage that uses the WebAudio API.
+
+To open the **WebAudio** tool:
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+
+1. In DevTools, on the main toolbar, click the **More Tools** (![More Tools icon.](../media/more-tools-icon-light-theme.png)) button, and then select **WebAudio**.
+
+   ![The WebAudio tool.](../media/webaudio-tool.png)
```

Comment:
This image does not show what the tool looks like when the page uses the WebAudio API. I'll try to provide another image in a minute.

Reply:
@captainbrosset - result: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/webaudio/webaudio-tool?branch=pr-en-us-1665